packages/lib/modules/price-impact/price-impact.utils.ts

```diff
@@ -87,3 +87,29 @@ export const getPriceImpactExceedsLabel = (priceImpactLevel: PriceImpactLevel) =
       return ''
   }
 }
+
+export function getPriceImpactLabel(priceImpact: string | number | null | undefined) {
+  return priceImpact
+    ? isZero(priceImpact)
+      ? ' (0.00%)'
+      : ` (-${fNum('priceImpact', priceImpact)})`
+    : ''
+}
```

Comment:
difficult to read such ternary conditions.. maybe smth like this?
```
export function getPriceImpactLabel(priceImpact: string | number | null | undefined): string {
  if (!priceImpact) {
    return '';
  }

  if (isZero(priceImpact)) {
    return ' (0.00%)';
  }

  return ` (-${fNum('priceImpact', priceImpact)})`;
}
```

Reply:
Good catch, I simply moved the function and forgot about refactoring. Done.